tpot/base.py

```diff
@@ -46,6 +46,7 @@
 from tqdm import tqdm
 from copy import copy, deepcopy
 
+import dask
```

Comment:
Please add installation of dask into [ci/.travis_install.sh](https://github.com/EpistasisLab/tpot/blob/master/ci/.travis_install.sh#L35-L36) and [.appveyor.yml](https://github.com/EpistasisLab/tpot/blob/master/.appveyor.yml#L22) for unit tests

Reply:
Done.  Though, just to reiterate, I'm not trying to get tests to work here at all.  This is only up here for conversation.

Reply:
OK, thanks. But it seems that it passed almost all the unit tests. Great!